CONTRIBUTING.md

```diff
@@ -25,6 +25,26 @@ We welcome small pull request contributions from anyone (docs improvements, bug
 - The maintainers will respond to your issue, please work with the maintainers to ensure that what you're doing is in scope for the project before writing any code.
 - If you have any doubt whether a contribution would be valuable, feel free to ask.
 
+## Developer Certificate of Origin
+
```

Comment:
Hey @AaronCrawfis - I'm not sure this is the best home for this. This [page](https://github.com/radius-project/radius/blob/main/docs/contributing/contributing-pull-requests/README.md) contains our instructions for sending a PR, and it feels like a real gap if DCO isn't a part of those instructions.

I'm OK with mentioning this in multiple places but I think it needs to be at least mentioned in the PR guide.

Reply:
Makes sense. Also included it in the PR guide below. Any other pages you'd like to see this added?